### PR TITLE
move /debug/info endpoint

### PR DESCRIPTION
**Pull request by mhilton**
Make the /debug/info endpoint sit on /debug/info rather than /v4/debug/info.

### CODE REVIEW

**Comment by mhilton** [comment]
## QA instructions

```
make server
http :8080/debug/info
```

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/903/
Test PASSed.

File: internal/charmstore/debug.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	"gopkg.in/juju/charmstore.v4/internal/router"
+	appver "gopkg.in/juju/charmstore.v4/version"
+)
+
+// GET /debug/info .
+func serveDebugInfo(http.Header, *http.Request) (interface{}, error) {
+	return appver.VersionInfo, nil
+}
+
+func newDebug() http.Handler {
```
Comment (urosj) [inline internal/charmstore/debug.go]: Could we call it newServiceDebugHandler() or something, making sure that it makes a difference from /vX/debug/status?

File: internal/charmstore/server.go [hunk 2]
```diff
@@ -70,6 +70,7 @@ func NewServer(db *mgo.Database, si *SearchIndex, config ServerParams, versions
 		}
 	})
 	mux := router.NewServeMux()
+	handle(mux, "/debug", newDebug())
```
Comment (urosj) [inline internal/charmstore/server.go]: Comment: // Version independent API.

**Comment by urosj** [comment]
:+1: With small suggestions.

File: internal/charmstore/debug.go (hunk 1 above)
Comment (mhilton) [inline internal/charmstore/debug.go]: done

File: internal/charmstore/server.go (hunk 2 above)
Comment (mhilton) [inline internal/charmstore/server.go]: done

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/904/
Test PASSed.

**Comment by jrwren** [comment]
:+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/905/
Test PASSed.

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://localhost:8080/job/charmstore-merge

**Pull request merged**